src/entities/LbpManager.ts

```diff
@@ -488,6 +489,7 @@ export class LbpManager implements ILaunch {
           .then((history) => {
             // this.ensureTrajectoryForecastData(history);
             this.priceHistory = history;
+            this.lastSwap = this.projectTokenHistoricalPriceService.lastSwap;
```

Comment:
why must this be here?  is it not easily accessible in the code where it is actually needed?

Reply:
The forecast data is being changed every time I attach the chart.
I need to ensure that the last swap I pass to it is the actual LBP last swap, otherwise, it uses the value initialized in the first initialization.

Reply:
removed from entity